### DISCUSSION TITLE
Can't remove licence

### DISCUSSION THREAD

**Discussion by Hakagi** [chr233/ASFEnhance#265] [提问求助 / Support] [answered]
I can't remove f2p games, I get this error everytime I try to.
![image](https://github.com/chr233/ASFEnhance/assets/65294828/9ed3e515-b332-4948-b9c6-9a6fd41c5dbe)

**Comment by woctezuma** [comment, marked as answer]
![firefox_fQRRZTxVdk](https://github.com/chr233/ASFEnhance/assets/570669/a2ac8ebb-11db-45c6-8982-dd79e2ee5841)
